integration-tests/artifacts/__tests__/index.js

```diff
@@ -328,6 +349,12 @@ function assertNodeCorrectness(runNumber) {
   })
 }
 
+function assertExitCode(runNumber) {
```

Comment:
What's the `runNumber` for in this function?

Reply:
It's currently not used as all builds are expected to succeed right now - I just used same structure as all the other assert helpers (`assertNodeCorrectness`, `assertHTMLCorrectness`, etc) that might have run specific assertions. I can drop it if you think this is cleaner

Reply:
Nah, I'm fine with it 👍